xcp_d/interfaces/qc_plot.py

```diff
@@ -283,7 +283,7 @@ def _run_interface(self, runtime):
 
         if self.inputs.tmask:  # If a tmask is provided, find # vols censored
             tmask_df = pd.read_table(self.inputs.tmask)
-            tmask_arr = tmask_df["framewise_displacement"]
+            tmask_arr = tmask_df["framewise_displacement"].values
```

Comment:
This fixes a bug introduced by recent PRs. Somehow those PRs' tests weren't catching it.